submit_cgap/scripts/submit_ontology.py

```diff
@@ -42,7 +48,22 @@ def main(simulated_args_for_testing=None):
                 validate_only=args.validate_only,
                 app=args.app,
                 submission_protocol=args.submission_protocol,
-        )
+                verbose=args.verbose, debug=args.debug)
+
+
+def verify_ontology_file(ontology_filename: str) -> bool:
+    if not os.path.exists(ontology_filename):
+        show(f"ERROR: Cannot find specified ontology (JSON) file: {ontology_filename}")
+        return False
```

Comment:
You should do `from dcicutils.command_utils import ScirptFailure` up top and then do (untested):
```suggestion
        raise ScriptFailure(f"ERROR: Cannot find specified ontology (JSON) file: {ontology_filename}")
```